src/libs/ajax.js

```diff
@@ -1249,7 +1250,8 @@ export const Ajax = signal => {
     CromIAM: CromIAM(signal),
     FirecloudBucket: FirecloudBucket(signal),
     OAuth2: OAuth2(signal),
-    Surveys: Surveys(signal)
+    Surveys: Surveys(signal),
+    WorkspaceDataService: WorkspaceDataService(signal)
```

Comment:
naming on this is not consistent with the other ajax sub-areas. 

I Suggest rename to WorkspaceData